### PR TITLE
Add to run_model_templates_tests.py possibility to load test ids from file

### PR DESCRIPTION
**Pull request by LeonidBeynenson**
Also add possibility to save the filtered list of tests to a file.

### CODE REVIEW

File: tests/run_model_templates_tests.py [hunk 1]
```diff
@@ -154,6 +156,24 @@ def _add_counter_to_rows(rows):
 
     return new_rows
 
+def save_list_to_path(all_tests, path):
```
Comment (Ilya-Krylov) [inline tests/run_model_templates_tests.py:159]: Could you remove `_to_path` and clarify which `list`

**Review by Ilya-Krylov — CHANGES_REQUESTED** [review]

File: tests/run_model_templates_tests.py (hunk 1 above)
Comment (LeonidBeynenson) [reply tests/run_model_templates_tests.py]: Fixed

**Comment by Ilya-Krylov** [comment]
run tests

**Comment by Ilya-Krylov** [comment]
@LeonidBeynenson please fix pylint issues

**Comment by LeonidBeynenson** [comment]
run tests

**Pull request merged**